src/Message.ts

```diff
@@ -254,3 +242,126 @@ export class MessageV2 extends MessageBase implements proto.MessageV2 {
 }
 
 export type Message = MessageV1 | MessageV2
+
+export interface DecodedMessage {
+  id: string
+  messageVersion: 'v1' | 'v2'
+  senderAddress: string
+  conversation: Conversation
+  contentType: ContentTypeId
+  sent: Date
+  contentTopic: string
+  recipientAddress?: string
+  error?: Error
+  content: any // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+export class DecodedMessage {
```

Comment:
How come you can export two things with the same name?

Reply:
Interfaces allow it (they become overloads), although I realized I don't really need the interface at all anymore.